applications/rag/example_notebooks/rag-data-ingest-with-kubernetes-docs.ipynb

```diff
@@ -0,0 +1,1835 @@
+{
```

Comment:
The notebook here includes all the outputs after executions along with the `execution_cout` got populated. Can you provide clean code of the notebook without execution output?